src/style.css

```
@@ -1 +1,85 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@1,200&family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
```

You can use shorthand property instead
https://developer.mozilla.org/en-US/docs/Web/CSS/padding#syntax
```suggestion
  padding: 0 50px;
```